Userland/Libraries/LibELF/DynamicLinker.cpp

```diff
@@ -226,6 +226,10 @@ static void initialize_libc(DynamicObject& libc)
     VERIFY(res.has_value());
     *((bool*)res.value().address.as_ptr()) = false;
 
+    res = libc.lookup_symbol("__auxiliary_vector"sv);
+    VERIFY(res.has_value());
+    *((void**)res.value().address.as_ptr()) = nullptr;
```

Comment:
Out of curiosity: Either this symbol exists, or it doesn't. If it doesn't exist, there's no real reason to `VERIFY(false)` here. If it does exist, it will be initialized when we call `__libc_init` a few lines down. Until then, nothing will access `__auxiliary_vector` anyway.

So, what does this code do?

Reply:
Right, no point in initializing it here :+1:

Reply:
(I also removed another redundant initialization in the DynamicLoader's `init_libc`).